src/Model/Serializer/AbstractSerializer.php

```diff
@@ -157,7 +157,7 @@ function (array $processedValue, ModelInterface $model): array {
                 },
                 []
             ),
-            MoneyType::TYPE_NAME => (int) round(bcmul((string)$value, '100.0', 1)),
+            MoneyType::TYPE_NAME => (int) round((float)bcmul((string)$value, '100.0', 1)),
```

Comment:
Can we trigger deprecation warning for this please? This should not used anymore